corehq/ex-submodules/pillowtop/feed/couch.py

```diff
@@ -19,12 +19,12 @@ def iter_changes(self, since, forever):
         from corehq.apps.change_feed.data_sources import SOURCE_COUCH
         extra_args = {'feed': 'continuous'} if forever else {}
         extra_args.update(self._extra_couch_view_params)
+        if self._couch_filter:
+            extra_args.update({'filter': self._couch_filter})
         self._last_processed_seq = since
         changes_stream = ChangesStream(
             db=self._couch_db,
-            heartbeat=True,
```

Comment:
Was this ever necessary, or did the default change when changing to cloudant?

Reply:
I don't think ```heartbeat``` was ever necessary, and cloudant now requires ```heartbeat``` to be an int.